doc/integrator/backend_qgis.rst

```diff
@@ -46,7 +46,8 @@ In the ``vars_<project>.yaml`` file, define:
 
 .. code:: yaml
 
-    mapserv_url: http://your.arcgis.server/arcgis/services/SOME/FOLDER/MapServer/WMSServer
+    mapserv_url: http://your.qgis.server/ows
+    geoserver: false
```

Comment:
There is a problem with this https://github.com/camptocamp/c2cgeoportal/blob/master/c2cgeoportal/scaffolds/update/CONST_vars.yaml_tmpl#L209 as it doesn't seem to be used. If "geoserver" is missing in the project vars file, log throw error like this:
```
[Wed Apr 27 15:54:20.006889 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503] Traceback (most recent call last):
[Wed Apr 27 15:54:20.006905 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/fanstatic/publisher.py", line 219, in __call__
[Wed Apr 27 15:54:20.006941 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return self.app(environ, start_response)
[Wed Apr 27 15:54:20.006960 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/fanstatic/injector.py", line 54, in __call__
[Wed Apr 27 15:54:20.006980 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = request.get_response(self.app)
[Wed Apr 27 15:54:20.007000 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/webob/request.py", line 1320, in send
[Wed Apr 27 15:54:20.007024 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     application, catch_exc_info=False)
[Wed Apr 27 15:54:20.007043 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/webob/request.py", line 1284, in call_application
[Wed Apr 27 15:54:20.007063 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     app_iter = application(self.environ, start_response)
[Wed Apr 27 15:54:20.007076 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/router.py", line 243, in __call__
[Wed Apr 27 15:54:20.007098 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = self.invoke_subrequest(request, use_tweens=True)
[Wed Apr 27 15:54:20.007118 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/router.py", line 218, in invoke_subrequest
[Wed Apr 27 15:54:20.007139 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = handle_request(request)
[Wed Apr 27 15:54:20.007156 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/tweens.py", line 21, in excview_tween
[Wed Apr 27 15:54:20.007200 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = handler(request)
[Wed Apr 27 15:54:20.007213 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 94, in tm_tween
[Wed Apr 27 15:54:20.007234 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     reraise(*exc_info)
[Wed Apr 27 15:54:20.007253 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid_tm/__init__.py", line 75, in tm_tween
[Wed Apr 27 15:54:20.007273 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = handler(request)
[Wed Apr 27 15:54:20.007290 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/router.py", line 164, in handle_request
[Wed Apr 27 15:54:20.007310 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = view_callable(context, request)
[Wed Apr 27 15:54:20.007329 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/c2cgeoportal/__init__.py", line 136, in new_f
[Wed Apr 27 15:54:20.007351 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return f(root, request)
[Wed Apr 27 15:54:20.007369 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/config/views.py", line 353, in rendered_view
[Wed Apr 27 15:54:20.007395 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     result = view(context, request)
[Wed Apr 27 15:54:20.007418 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/pyramid/config/views.py", line 481, in _class_requestonly_view
[Wed Apr 27 15:54:20.007443 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     response = getattr(inst, attr)()
[Wed Apr 27 15:54:20.007462 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/c2cgeoportal/views/entry.py", line 915, in get_cgxp_viewer_vars
[Wed Apr 27 15:54:20.007490 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     themes, errors = self._themes(role_id, interface)
[Wed Apr 27 15:54:20.007510 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 1040, in decorate
[Wed Apr 27 15:54:20.007536 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     should_cache_fn)
[Wed Apr 27 15:54:20.007555 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 651, in get_or_create
[Wed Apr 27 15:54:20.007581 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     async_creator) as value:
[Wed Apr 27 15:54:20.007599 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/core/dogpile.py", line 158, in __enter__
[Wed Apr 27 15:54:20.007621 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return self._enter()
[Wed Apr 27 15:54:20.007639 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/core/dogpile.py", line 98, in _enter
[Wed Apr 27 15:54:20.007669 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     generated = self._enter_create(createdtime)
[Wed Apr 27 15:54:20.007689 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/core/dogpile.py", line 149, in _enter_create
[Wed Apr 27 15:54:20.007713 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     created = self.creator()
[Wed Apr 27 15:54:20.007731 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 619, in gen_value
[Wed Apr 27 15:54:20.007757 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     created_value = creator()
[Wed Apr 27 15:54:20.007776 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/dogpile/cache/region.py", line 1036, in creator
[Wed Apr 27 15:54:20.007796 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     return fn(*arg, **kw)
[Wed Apr 27 15:54:20.007815 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]   File "/var/www/vhosts/gmfusrgrp_version2-geomapfishtest/private/yjn/test/.build/venv/lib/python2.7/site-packages/c2cgeoportal/views/entry.py", line 589, in _themes
[Wed Apr 27 15:54:20.007836 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503]     role_id if self.mapserver_settings["geoserver"] else None
[Wed Apr 27 15:54:20.007865 2016] [wsgi:error] [pid 12168] [remote 81.18.187.187:19503] KeyError: 'geoserver'
```